### PR TITLE
add and remove items.

### PR DESCRIPTION
**Pull request by Bifabig**
- Removed all hardcoded items from the tasks array.
- Implemented a function for adding a new task.
- Implemented a function for deleting a task.
- Implemented a function for editing task descriptions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,247 @@
...
+
+I would like to thank Microverse and my coding partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (wayungi) [inline README.md:181]: - [x] Licence link is broken. Please add a license to the project and set the correct path 😄

File: README.md [hunk 2]
```diff
@@ -1 +1,247 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- > Add a link to your deployed project. -->
+
+- <a href="#">Live Demo</a>
```
Comment (wayungi) [inline README.md]: - [x] Live demo link leads nowhere. Kindly deploy the project & update this link or delete all together if deployment of the project is not a requirement 🤝

File: README.md [hunk 3]
```diff
@@ -1 +1,247 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To Do List <a name="about-project"></a>
+
+<!-- > Describe your project in 1 or 2 sentences. -->
```
Comment (wayungi) [inline README.md]: - [ OPTIONAL] Please delete all these comments. Their role is to only guide you in filling your readme. The should be deleted once the document is fully edited 👏🏾

File: README.md [hunk 4]
```diff
@@ -1 +1,247 @@
...
+```
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
```
Comment (wayungi) [inline README.md]: - [ OPTIONAL] Please delete all commnets once you have finished editing your `readme`

File: src/functions.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-use-before-define */
```
Comment (wayungi) [inline src/functions.js]: [ ]  Please do not disable linters. Linters help us write good-quality code. 👍🏾

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  if (e.key === 'Enter') {
+    addTodo(e.target.value);
+    renderTodos();
+    e.target.value = '';
+  }
+});
+
+// document.querySelector('.clear-btn').addEventListener('click', () => {
+//   deleteAll();
+// });
```
Comment (wayungi) [inline src/index.js]: - [x] Please remove all unused code that is commented. Commented code makes your repo dirty. 😃

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Bifabig , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks are passing ✔️ 
- [ ] Good commit messages ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @Bifabig , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**